### DISCUSSION TITLE
[Bug]: The conversation list cannot see new conversations

### DISCUSSION THREAD

**Discussion by Passerby1011** [danny-avila/LibreChat#2405] [Troubleshooting]
### What happened?

After you delete a conversation, simply start a new conversation in the input box (do not click New chat). At this point, the conversation list will not show the new conversation until you refresh it.

### Steps to Reproduce

1. Delete an existing conversation, and your conversation will be cleared.
2. Do not click "New chat"; instead, engage in conversation directly within the dialogue box.
3. You'll notice that you've created a new conversation at this point, but you won't find it in the conversation list until you refresh the page

### Screenshots

![image](https://github.com/danny-avila/LibreChat/assets/63802758/e3c45e6d-350e-4049-b503-7c0db1c33558)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Thanks for reproducing and reporting this. It will be an easy fix.

**Comment by danny-avila** [comment]
Addressed by https://github.com/danny-avila/LibreChat/pull/2414
